scripts/update_grafana.py

```diff
@@ -0,0 +1,49 @@
+import json
+import logging
+import subprocess
+from requests.auth import HTTPBasicAuth
+import requests
+
+from azul import config
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    logging.basicConfig(format='%(asctime)s %(levelname)-7s %(threadName)-7s: %(message)s',
+                        level=logging.INFO
+                        )
```

Comment:
Don't invent new wrapping conventions. With aligned indent the closing brace/parenthesis/bracket goes on the same line, not a new line.